Docker/Dockerfile

```diff
@@ -118,7 +118,7 @@ RUN if [ "${IMAGE_TYPE}" = "simulation" ]; then \
         $WORKSPACE/o3de/scripts/o3de.sh register -pp $WORKSPACE/o3de-demo-project/  && \
         . /opt/ros/${ROS_VERSION}/setup.sh && \
         cmake -B $WORKSPACE/o3de-demo-project/build/linux -S $WORKSPACE/o3de-demo-project -G "Ninja Multi-Config" -DLY_STRIP_DEBUG_SYMBOLS=TRUE -DLY_DISABLE_TEST_MODULES=ON && \
-        cmake --build $WORKSPACE/o3de-demo-project/build/linux --config profile --target AssetProcessorBatch ROS2-Gem-Demo.GameLauncher ROS2-Gem-Demo.Assets -j 20 && \
+        cmake --build $WORKSPACE/o3de-demo-project/build/linux --config profile --target AssetProcessorBatch ROS2-Gem-Demo.GameLauncher RobotVacuumSample.Assets -j 20 && \
```

Comment:
```suggestion
        cmake --build $WORKSPACE/o3de-demo-project/build/linux --config profile --target AssetProcessorBatch RobotVacuumSample.GameLauncher RobotVacuumSample.Assets -j 20 && \
```

Reply:
More question, hardcoded 20 threads can lead to exhaustion of memory, maybe be more conservative here?

Reply:
Lets remove it and allow ninja to use its default. This is only a problem on high-core low memory machines